pkg/ccloudv2/networking.go

```diff
@@ -31,3 +31,10 @@ func (c *Client) DeleteNetwork(envId, id string) error {
 	httpResp, err := c.NetworkingClient.NetworksNetworkingV1Api.DeleteNetworkingV1Network(c.networkingApiContext(), id).Environment(envId).Execute()
 	return errors.CatchCCloudV2Error(err, httpResp)
 }
+
+func (c *Client) UpdateNetwork(envId, id string, update networkingv1.NetworkingV1NetworkUpdate) (networkingv1.NetworkingV1Network, error) {
+	update.Spec.SetEnvironment(networkingv1.ObjectReference{Id: envId})
```

Comment:
Can you set this outside of this function?

Reply:
Yeah, saw this pattern somewhere so I though this was the convention.

Reply:
Making a note to delete that pattern, thanks!